Allow SVG transformation

rudashi opened the pull request:
This change enables the user to transform the svg before it's processed by the component.

It's different approach for @econic solution.

rudashi committed:
Allow form SVG transformation

This change enables the user to transform the svg before it's processed by the component.

It's different approach for @econic solution.

shrpne commented:
Nice!

Can you add corresponding prop to the component and some docs about it, please?

rudashi committed:
Add missing prop with docs

This example create circle in svg:

```
<inline-svg :src="require(`./../../assets/svg/${computedMap.image}`)" :transformSource="transform"/>

....

const transform = (svg) => {
	let point = document.createElementNS("http://www.w3.org/2000/svg", 'circle');
        point.setAttributeNS(null, 'cx', '1376.7');
        point.setAttributeNS(null, 'cy', '565.8');
        point.setAttributeNS(null, 'r', '10');
        point.setAttributeNS(null, 'fill', 'red');
        svg.appendChild(point);
	return svg;
}
```
For cleaner syntax you could use https://github.com/svgdotjs/svg.js

shrpne commented:
Thank you!

Published under [v1.2.0](https://github.com/shrpne/vue-inline-svg/releases/tag/v1.2.0) tag
